src/components/StripeTerminalProvider.tsx

```diff
@@ -176,6 +179,18 @@ export function StripeTerminalProvider({
     [log]
   );
 
+  const didStartReaderReconnect = useCallback(() => {
+    log('didStartReaderReconnect');
+  }, [log]);
+
+  const didSucceedReaderReconnect = useCallback(() => {
+    log('didSucceedReaderReconnect');
+  }, [log]);
+
+  const didFailReaderReconnect = useCallback(() => {
+    log('didFailReaderReconnect');
+  }, [log]);
+
```

Comment:
should emit the error message here

Reply:
example https://github.com/stripe/stripe-terminal-react-native/blob/1870117c/src/components/StripeTerminalProvider.tsx#L106